bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -243,7 +243,11 @@ public String toString() {
      */
     @Override
     public void stateChanged(Item item, State oldState, State newState) {
+        State oldGroupState = this.state;
         setState(function.calculate(members));
+        if (!oldGroupState.equals(this.state)) {
+            sendGroupStateChangedEvent(item.getName(), this.state, oldGroupState);

```

Comment:
> But IMHO an empty method would lead to an unexpected behavior for users.

In what sense? An empty method only means that we do not apply any further logic if the member state has changed (and not only updated). And this is the case: For the GroupItem, we really do not care.